build-cake.ps1

```diff
@@ -0,0 +1,5 @@
+# Hack until dotnet/cli##1701 is accepted
```

Comment:
Could we go back to `build.ps1` and `build.sh`, I know underlying they will bootstrap cake for me, but from the outside I don't really care that cake is the build system, I just want to `build`.  Feels like an extra 5 characters for nothing (I know I know tab completion etc :dragon_face: )